tests/plugins/__snapshots__/connectionPlugin.spec.ts.snap

```diff
@@ -200,7 +200,7 @@ exports[`basic behavior should adhere to the Relay spec 2`] = `
   cursor: String!
 
   \\"\\"\\"https://facebook.github.io/relay/graphql/connections.htm#sec-Node\\"\\"\\"
-  node: User
```

Comment:
I'm wondering why these snapshots were incorrect in the first place. This definitely does not adhere to the Relay Connection specification. There's also a very good reason behind the design choices in the spec.

```gql
edges: [User!]!
...
cursor: String!
node User!
```

The above is bad because an issue with a node, e.g. a required field resolves to `null`, results in a null value propagating up to the first non-nullable field. And because edges can't have null elements and edges themselves can't be null, one broken node would break the entire connection.

```gql
edges: [User!]!
...
cursor: String!
node User
```

In this case, node can be nullable but then a non-nullable cursor doesn't make any sense. `cursors` are usually derived from the node and must be unique in the connection. How does one derive the cursor for a `null` and what if there are 2 `null` nodes in the result set?

The only configuration that makes sense is the specified:
```gql
edges: [User]
...
cursor: String!
node User!
```
possibly with edges being `[User]!` in my opinion.